### ISSUE TITLE
What if the primary action when viewing a profile was to make a 'connection request', but if you're viewing a profile of someone you're connected to, then the primary action is to message them?

### ISSUE THREAD

**Issue by bsteinbeigle** [Methodician/guytime#95]

**Comment by bsteinbeigle** [comment]
@kaxline I can see your point of view. I believe a hybrid approach may work best. For example when sending someone a connection request, the person sending the request should have the option to send the other person a short message. What if someone shared a mutual friend for example and wanted to mentioned that when reaching out initially. The goal here is to make it as easy as possible for two people to strike up a conversation addressing the human laziness factor! Having email notifications will certainly help;)

**Comment by kaxline** [comment]
@bsteinbeigle Yeah, I was thinking the request could contain a message, and if it's accepted, maybe it just starts a chat like it does now. We'd just be adding the ability to ignore or decline a request ...

**Comment by bsteinbeigle** [comment]
@kaxline Ya that sounds great! Its good to give someone the opportunity to strike up a conversation at least.
